### PR TITLE
test: add unit test for util.go `makeSecureTempDir` function

### PR DESCRIPTION
**Pull request by mirageoasis**
#19303  , this PR deals with test for `makeSecureTempDir` function 

Summary

This PR tests unit test for `util.go` `makeSecureTempDir` function.

### CODE REVIEW

File: commitserver/commit/util_test.go [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+	"os"
+	"testing"
+)
+
+func TestMakeSecureTempDir(t *testing.T) {
+	tempDir, cleanup, err := makeSecureTempDir()	
+	
+	if err != nil {
+		t.Fatalf("Failed to create secure temporary directory: %v", err)
+	}
```
Comment (crenshaw-dev) [inline commitserver/commit/util_test.go:14]: Instead of `t.Fatalf`, could you switch to `require.NoError`?

**Review by crenshaw-dev — CHANGES_REQUESTED** [review]
Thanks for the PR!! Requested one small change.

File: commitserver/commit/util_test.go (hunk 1 above)
Comment (mirageoasis) [reply commitserver/commit/util_test.go]: Done with requests!

**Review by crenshaw-dev — APPROVED** [review]
Thanks!

**Pull request merged**